docker-compose.yml

```diff
@@ -1,10 +1,9 @@
 services:
   msls:
     build: ./dev-local
-    command: bash -c "python -m debugpy --listen 0.0.0.0:7890 app.py"
```

Comment:
Thank god. This has been an intermittent headache. Love to see it go.

Reply:
yea not required as per original spike